From a7d309481839b06b375ec89160f090d1d10735ff Mon Sep 17 00:00:00 2001
From: Athanasios Oikonomou <athoik@gmail.com>
Date: Fri, 12 Jul 2013 19:47:01 +0300
Subject: [PATCH 1/1] Fix ac3 audio playback if aac track is present

Once an aac track was played back, an aac adts header was added to
pes packet regardless of whether aac track is selected or not.

Thanks to betacentauri.
---
 src/gstdvbaudiosink.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gstdvbaudiosink.c b/src/gstdvbaudiosink.c
index 93cb587..1a3824e 100644
--- a/src/gstdvbaudiosink.c
+++ b/src/gstdvbaudiosink.c
@@ -605,6 +605,7 @@ gst_dvbaudiosink_set_caps (GstBaseSink * basesink, GstCaps * caps)
 
 	self->skip = 0;
 	self->block_align = 0;
+	self->aac_adts_header_valid = FALSE;
 
 	if (!strcmp(type, "audio/mpeg")) {
 		gint mpegversion;
-- 
1.7.9.5

